### ISSUE TITLE
Another Option to Consider

### ISSUE THREAD

**Issue by BrandonR541** [bakkeby/choosing-a-sync-service-under-linux#2]
Amazing work, thank you!
Another fantastic option is FolderSync, definitely worth having on the list I think it's pretty great. Earlier this year they said that they would stop supporting the desktop version, but now it seems like they're back to fully continue finishing and supporting the desktop products for Windows iOS and Linux, in addition to  iPhone and Android.  Lots of good syncing capabilities and works across a bunch of different cloud storage providers as well as local file systems and local or custom FTP SMB etc

https://foldersync.io/

**Comment by bakkeby** [comment]
I get the impression that this is primarily targeted at Android and iPhone users. Seems fairly expensive. With a licence price of €25 per _device_ it can stack up pretty quickly, and you need to renew these after two years if you want to get updates.

It makes sense if you have a mobile phone and want to have a way to back data up on a NAS or a cloud service. Perhaps not so much if you have five Linux installations that you want to keep in sync.

As far as I can tell they don't do any form of hosting; they just piggy-back on other file hosting services and/or local hosting. Functionality wise it looks very capable with many options.

I wasn't able to get the linux client running, but I presume that one may have better luck on Ubuntu which this is made for.

**Comment by BrandonR541** [comment]
Thanks for the response, and just want to say again how awesome your work
is and how much I appreciate it.

I definitely agree that it is more targeted at mobile users, at one point
they didn't even have a desktop version, and for having a bunch of devices
and keeping the most updated version it is pretty expensive, but there may
be some use cases that fit well. Definitely goes in a different category as
like you mentioned it does not host its own files it's more of a tool for
managing multiple cloud accounts plus more control and capability for file
sync on mobile devices for people who already have cloud storage accounts
from Google Dropbox etc.

I found it useful and at one point it was the only thing I personally could
find to do what I needed in that specific instance, and after coming across
your GitHub page I thought others might find a use for it as well so I
wanted to share with you just in case :)


I'm an Android user and that's how I originally found it, at that time
there was no desktop version only the mobile app, and the reason I liked it
so much was its ability to sync between different cloud accounts, while
having decent configurability and such. I had a bunch of files on Google
drive, OneDrive, and Dropbox and I was trying to get things straightened
out and had a hard time finding a solution.

I first started using it I think early last year and, at least at that
time, there were no good options for syncing between different cloud
accounts (for free) (that I could find) other than some random Chrome
extensions or unknown websites that were all questionable security-wise and
that's why it stood out to me.

You can freely use the mobile app including syncing local device files, as
well as files already in the cloud from other providers, between different
cloud accounts. As far as I can tell only the desktop version is paid.
Actually I think I might have gotten the pro version of the Android app for
just a few dollars one time purchase, but if I remember correctly it wasn't
because the features I needed were behind a paywall I just wanted to show
appreciation to the devs, but I may be wrong.


I was able to get the desktop version running on Linux yesterday, I'm on an
arch derivative called EndeavorOS and I just extracted the .deb from their
download page and ran the executable and it seemed to work just fine. I
have not used the desktop version extensively yet, I'm also setting up
syncthing and still figuring out what I want my solution to be there.


It's not for everyone, and there are many other great solutions, but I do
really like the mobile app for managing mobile device files and managing
multiple cloud accounts from other providers that you're already paying for
but are isolated from each other. That's where it shines and, in my
opinion, the fact that they now have desktop app is nice (ignoring the
pricing), but since there are already apps from the normal cloud providers
for desktop I think it's more of a bonus feature for mobile focused users
to have another desktop tool if desired.


Anyways, I just wanted to share with you because your amazing research is
so thorough and extensive and I thought it was worth considering this
missing option, it might not fit the criteria for your list, and I may be
getting some things incorrect that I've shared with you because I'm not an
expert, but I just wanted to Let you know it exists even if it doesn't end
up on your list, and really just wanted to let you know I appreciate your
hard work and efforts to help other figure out their options.

Have a great day, and a wonderful new year!

Brandon

**Comment by bakkeby** [comment]
I very much appreciate it Brandon. Always interesting to learn about new solutions that I may have missed.

I suppose a lot has changed as well since I looked into this five years ago :) I had a similar struggle trying to find a note taking application that fits my needs. Mind map applications are also fairly limited.

I added FolderSync to the list - in the context of what I was originally looking for. The list is not ordered in a particular way.

I'll just mark this issue as closed, but feel free to share if you have anything more to add.
